components/ingest-service/grpc/grpc.go

```diff
@@ -124,7 +124,7 @@ func Spawn(opts *serveropts.Opts) error {
 
 	// ChefRuns
 	chefIngest := server.NewChefIngestServer(client, authzProjectsClient, nodeMgrServiceClient,
-		opts.MaxNumberOfBundledRunMsgs, opts.MaxNumberOfBundledActionMsgs)
+		opts.MaxNumberOfBundledRunMsgs, opts.MaxNumberOfBundledActionMsgs, opts.NumberOfRunMsgsTransformers)
```

Comment:
would it make sense to start passing the entire `opts` struct? My rule is 3 or more parameters from the same struct it is worth passing the entire pointer and use it internally inside the function.